src/Compressor/CssMinCompressor.php

```diff
@@ -18,27 +13,24 @@ public function __construct()
 
     protected function execute(string $string): string
     {
-        // If there's no selector, this must be an inline CSS attribute
-        if (strlen($string) > 0 && strpos($string, '{') === false) {
+        // If there's no selector, this must be an inline CSS attribute.
+        if (
+            \strlen($string) > 0
+            &&
+            \strpos($string, '{') === false
+        ) {
             return $this->minifyInline($string);
         }
 
-        return $this->cssMin->minify($string);
+        return $this->cssMin::minify($string);
     }
 
-    /**
-     * Minifies inline CSS attributes.
-     *
-     * @param string $string
-     *
-     * @return string
-     */
-    private function minifyInline($string): string
+    private function minifyInline(string $string): string
     {
-        // Get CssMin to compress inline CSS by adding and stripping a selector
+        // Get CssMin to compress inline CSS by adding and stripping a selector.
         $mock = 'body{' . $string . '}';
         $minified = $this->execute($mock);
 
-        return substr($minified, 5, -1);
+        return \substr($minified, 5, -1);
```

Comment:
Awesome! Been working for the past few days updating all my packages to do this, so this change is very welcome :+1: !!!